### ISSUE TITLE
Glass Textures are fliped horizonally

### ISSUE THREAD

**Issue by GalaxyThePanda** [Chocohead/OptiFabric#1164]
The textures for glass have been flipped
![2023-08-03_14 31 04](https://github.com/Chocohead/OptiFabric/assets/141367821/ee115e8f-252f-4c61-87bf-d54fcc4b8195)

**Comment by ABradyK** [comment]
I noticed this issue too. It's seems like they only flip on the south and east sides. 
![image](https://github.com/Chocohead/OptiFabric/assets/123845946/a23cc4f5-3179-4759-abf4-1b1a073f71b1)
Here's a screenshot.

**Comment by RU-B0** [comment]
Just wanted to comment how I fixed this issue: I found out that a mod called fusion (https://modrinth.com/mod/fusion-connected-textures) launches with OptiFine just fine. Using Midnighttigger's Default Connected Textures (https://modrinth.com/resourcepack/mt-ct-d/version/tfKvt8nl) and putting it as lowest priority on the resource pack list (but not below MC itself or fabric), it gets rid of this issue. Plus, OptiFine's implementation still works for other resource packs. Its a win - win. 

Tested with Fusion 1.0.5a and Optifine HD U I5, Optifabric 1.13.25, and FAPI 0.83.1 (+10000000000 other mods but you get the point)
